geospaas_harvesting/ingesters.py

```diff
@@ -539,15 +540,30 @@ def _get_normalized_attributes(self, dataset_info, *args, **kwargs):
         # set optional ForeignKey metadata from Nansat or from defaults
         normalized_attributes['gcmd_location'] = n_metadata.get(
             'gcmd_location', pti.get_gcmd_location('SEA SURFACE'))
-        normalized_attributes['provider'] = n_metadata.get(
-            'data_center', pti.get_gcmd_provider('NERSC'))
+        normalized_attributes['provider'] = pti.get_gcmd_provider(
+            n_metadata.get('provider', 'NERSC'))
         normalized_attributes['iso_topic_category'] = n_metadata.get(
             'ISO_topic_category', pti.get_iso19115_topic_category('Oceans'))
 
         # Find coverage to set number of points in the geolocation
         if len(nansat_object.vrt.dataset.GetGCPs()) > 0:
             nansat_object.reproject_gcps()
         normalized_attributes['location_geometry'] = GEOSGeometry(
-            nansat_object.get_border_wkt(nPoints=n_points), srid=4326)
+            nansat_object.get_border_wkt(n_points=n_points), srid=4326)
+        json_dumped_dataset_parameters = n_metadata.get('dataset_parameters', None)
+        if json_dumped_dataset_parameters:
+            try:
+                json_loads_result = json.loads(json_dumped_dataset_parameters)
+            except TypeError:
+                self.LOGGER.error("'dataset_parameters' section of metadata is incorrectly dumped.")
+            else:
+                if isinstance(json_loads_result, list):
+                    normalized_attributes['dataset_parameters'] = [
+                        get_cf_or_wkv_standard_name(dataset_param)
```

Comment:
Have you considered what happens if the parameter is not found in pythesint?

Reply:
in the metanorm we have this implementation of this function:
![image](https://user-images.githubusercontent.com/62345618/105986029-9e2f5980-609c-11eb-9664-f628ec0d3fb6.png)
Here I think I should raise an error instead of passing.right?

Reply:
It depends on which behavior you want here.
If you want to interrupt the ingestion of the current dataset when an unknown parameter is found, then leave it as it is.
If you want to still add the current dataset, ignoring unknown parameters, then you need to catch the exception (in that case, at least log a warning or error message).

Reply:
We want to interrupt,because there is not point for us to develop an uncomplete mapper. This local harvester works with mappers or gdal. If a mapper write for `dataset_parameter` section of it. It must be found in the standard names. otherwise that developement of mapper is pointless.

Reply:
so I leave that single line as its current state.

Reply:
Keep in mind that the dataset parameters might not always be hard coded in mappers.
Maybe some mappers will extract the parameter names from the dataset files. In that case, those names might not always be standard.
Anyway, I don't have a strong opinion on the matter, I am just making sure you take that case into account when you make your decision.

Reply:
Yes, that is the point! the developer of mapper must deliver it in a standard way! only standard one. otherwise, the development is useless because this code and this PR is generic. The nature of mapper code is specific,so the standard name of each parameter should delivered specifically there.